lib/gds_api/test_helpers/asset_manager.rb

```diff
@@ -10,6 +10,13 @@ def asset_manager_has_an_asset(id, atts)
           .to_return(body: response.to_json, status: 200)
       end
 
+      def asset_manager_has_whitehall_asset(legacy_url_path, atts)
```

Comment:
Minor: For consistency with `asset_manager_has_an_asset ` and `asset_manager_does_not_have_a_whitehall_asset `, `asset_manager_has_whitehall_asset` should be `asset_manager_has_a_whitehall_asset`.